lib/ts/recipe/passwordless/components/features/signInAndUp/index.tsx

```diff
@@ -264,81 +267,112 @@ export const SignInUpFeature: React.FC<
 export default SignInUpFeature;
 
 function getModifiedRecipeImplementation(
-    originalImpl: RecipeInterface,
+    recipe: Recipe,
```

Comment:
You should pass the recipe implementation here. And call it `originalImpl`. Just to minimise the number of changes.